tests/modules/graphql_persisted_queries_test/src/Plugin/GraphQL/PersistedQuery/PersistedQueryPluginOne.php

```diff
@@ -25,6 +25,9 @@ public function getQuery($id, OperationParams $operation) {
     return $queryMap[$id] ?? NULL;
   }
 
+  /**
+   * @cover queryMap::resolve
+   */
```

Comment:
`@cover` annotation says which part of the code is covered by the tests in the current method. We shouldn't use it here. Here we are fixing `Drupal.Commenting.FunctionComment.Missing` rule which should be fixed by commenting the method instead. Let's  just use:
```
  /**
   * Map between persisted query IDs and corresponding GraphQL queries.
   */
```